cpg_infra/billing_aggregator/monthly_aggregate/main.py

```diff
@@ -145,4 +207,4 @@ def get_billing_data(invoice_month: str) -> DataFrame:
     logging.getLogger('urllib3').setLevel(logging.WARNING)
     event_loop = asyncio.new_event_loop()
 
-    event_loop.run_until_complete(process_and_upload_monthly_billing_report())
+    # event_loop.run_until_complete(process_and_upload_monthly_billing_report(None))
```

Comment:
Is this intentionally commented out? Should still work in cloud function, but you couldn't run it locally right?

Reply:
Yup unintentional